tlsutil/config.go

```diff
@@ -49,6 +49,10 @@ type Config struct {
 	// existing clients.
 	VerifyServerHostname bool
 
+	// UseTLS is used to enable outgoing TLS verification of all outgoing connections to
```

Comment:
This comment is a little misleading with the word "verification". I'd just say "UseTLS enables outgoing TLS connections to Consul servers."